salt/_modules/containerd.py

```diff
@@ -7,7 +7,7 @@
 
 import salt.utils.path
 
-log = logging.getLogger(__name__)
+LOG = logging.getLogger(__name__)
```

Comment:
Most other (upstream) Salt modules use `log` i.s.o. `LOG`. Maybe this should be an exception in the linter.

Reply:
On the other hand, it's pretty common to have UPPERCASE for global variable.

So depends if we want clean code or Salt-like code here.

But if we add an exception to the linter, it should be only for Salt code, as we probably don't want these kind of weird decision to be applied to Python script in general

Reply:
Leaving it as `log` for now since I'm dropping Python linting

Reply:
Why reverting that change? You'll have to reintroduce it when we re-enable linting anyway, so you could keep it.

Reply:
It's trivial enough to reproduce, and if I want to see if I can work in the Salt-specific exception.